includes/fbutils.php

```diff
@@ -729,8 +729,14 @@ public static function get_cached_best_tip() {
 		 * @return array
 		 */
 		public static function normalize_product_data_for_items_batch( $data ) {
-			// Allowed values are 'refurbished', 'used', and 'new', but the plugin has always used the latter.
-			$data['condition'] = 'new';
+			/*
+			 * To avoid overriding the condition value, we check if the value is set or is not one of
+			 * the allowed values before setting it to 'new'. Allowed values are 'refurbished', 'used', and 'new'.
+			 */
+			if ( ! isset( $data['condition'] ) || ! in_array( $data['condition'], [ 'refurbished', 'used', 'new' ], true ) ) {
```

Comment:
Short array syntax is disallowed in WooCommerce core but allowed in this repo. We might want to take a call here if we wish to align our code with the core going forward.

Reply:
Fixed this here. but you're correct, short array syntaxes are used throughout the code base. We can create a separate issue to align coding styles.